core/code/entity_decode.js

```diff
@@ -111,9 +109,23 @@ function detailsPortalData(a) {
   }
 };
 
-function extendedPortalData(a, required) {
+// on getEntities, intel send the history unless the user has never interacted with the portal
+// raw data is either a summary or an extended data with null details
+// in the first case, history is forced to 0 if missing
```

Comment:
It is unclear what do you mean by 'first case'.
Please change the wording.

Reply:
```suggestion
// when it is a summary, history is forced to 0
```